apps/accounts/views.py

```diff
@@ -0,0 +1,31 @@
+from django.conf import settings
+from django.contrib.auth import login
+
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from waffle.decorators import waffle_switch
+
+from . import verify
+from users.models import UserProfile
+
+
+class LoginView(APIView):
+
+    @waffle_switch('fxa-auth')
+    def post(self, request):
+        if 'code' not in request.DATA:
+            return Response({'error': 'No code provided.'}, status=400)
+
+        identity = verify.fxa_identify(request.DATA['code'],
+                                       config=settings.FXA_CONFIG)
+        email = identity.get('email')
+        if email:
+            try:
+                user = UserProfile.objects.get(email=email)
+            except UserProfile.DoesNotExist:
+                return Response({'error': 'User does not exist.'}, status=400)
+            else:
```

Comment:
Stats are different from logging. Stats give us insight into performance at an aggregate level. Logging would let us debug why a certain user is having trouble logging in.